api/app/schemas/hfi_calc.py

```diff
@@ -144,6 +144,12 @@ class StationInfo(BaseModel):
     fuel_type_id: int
 
 
+class HFILoadResultRequest(BaseModel):
+    """ Request to load the HFI Calculator. """
+    start_date: Optional[date]
```

Comment:
Just want to confirm for my own understanding - if a fire centre has a prep sheet stored in our DB where the prep sheet's `start_date` matches this `start_date`, the result is returned? Or is it that any date in the prep sheet's prep period that includes this `start_date` returns a result?

Reply:
Not period - has to match the start date.